lib/inquisition/outputter/docx/doc_view_helper.rb

```diff
@@ -2,9 +2,29 @@ module Inquisition
   module Outputter
     module Docx
       class DocViewHelper
+        ERROR_FIND_GEMS_MESSAGE = 'There is no one gem'.freeze
```

Comment:
Maybe we should create a separate class which will be responsible for `technology stack` data, what do you think?